api/estados/santacatarina.js

```diff
@@ -1,300 +1,302 @@
+    'Leoberto Leal',
+    'Lindóia do Sul',
+    'Lontras',
+    'Luiz Alves',
+    'Luzerna',
+    'Macieira',
+    'Mafra',
+    'Major Gercino',
+    'Major Vieira',
+    'Maracajá',
+    'Maravilha',
+    'Marema',
+    'Massaranduba',
+    'Matos Costa',
+    'Meleiro',
+    'Mirim Doce',
+    'Modelo',
+    'Mondaí',
+    'Monte Carlo',
+    'Monte Castelo',
+    'Morro da Fumaça',
+    'Morro Grande',
+    'Navegantes',
+    'Nova Erechim',
+    'Nova Itaberaba',
+    'Nova Trento',
+    'Nova Veneza',
+    'Novo Horizonte',
+    'Orleans',
+    'Otacílio Costa',
+    'Ouro',
+    'Ouro Verde',
+    'Paial',
+    'Painel',
+    'Palhoça',
+    'Palma Sola',
+    'Palmeira',
+    'Palmitos',
+    'Papanduva',
+    'Paraíso',
+    'Passo de Torres',
+    'Passos Maia',
+    'Paulo Lopes',
+    'Pedras Grandes',
+    'Penha',
+    'Peritiba',
+    'Petrolândia',
+    'Piçarras',
+    'Pinhalzinho',
+    'Pinheiro Preto',
+    'Piratuba',
+    'Planalto Alegre',
+    'Pomerode',
+    'Ponte Alta',
+    'Ponte Alta do Norte',
+    'Ponte Serrada',
+    'Porto Belo',
+    'Porto União',
+    'Pouso Redondo',
+    'Praia Grande',
+    'Presidente Castelo Branco',
+    'Presidente Getúlio',
+    'Presidente Nereu',
+    'Princesa',
+    'Quilombo',
+    'Rancho Queimado',
+    'Rio das Antas',
+    'Rio do Campo',
+    'Rio do Oeste',
+    'Rio do Sul',
+    'Rio dos Cedros',
+    'Rio Fortuna',
+    'Rio Negrinho',
+    'Rio Rufino',
+    'Riqueza',
+    'Rodeio',
+    'Romelândia',
+    'Salete',
+    'Saltinho',
+    'Salto Veloso',
+    'Sangão',
+    'Santa Cecília',
+    'Santa Helena',
+    'Santa Rosa de Lima',
+    'Santa Rosa do Sul',
+    'Santa Terezinha',
+    'Santa Terezinha do Progresso',
+    'Santiago do Sul',
+    'Santo Amaro da Imperatriz',
+    'São Bento do Sul',
+    'São Bernardino',
+    'São Bonifácio',
+    'São Carlos',
+    'São Cristovão do Sul',
+    'São Domingos',
+    'São Francisco do Sul',
+    'São João Batista',
+    'São João do Itaperiú',
+    'São João do Oeste',
+    'São João do Sul',
+    'São Joaquim',
+    'São José',
+    'São José do Cedro',
+    'São José do Cerrito',
+    'São Lourenço do Oeste',
+    'São Ludgero',
+    'São Martinho',
+    'São Miguel da Boa Vista',
+    'São Miguel do Oeste',
+    'São Pedro de Alcântara',
+    'Saudades',
+    'Schroeder',
+    'Seara',
+    'Serra Alta',
+    'Siderópolis',
+    'Sombrio',
+    'Sul Brasil',
+    'Taió',
+    'Tangará',
+    'Tigrinhos',
+    'Tijucas',
+    'Timbé do Sul',
+    'Timbó',
+    'Timbó Grande',
+    'Três Barras',
+    'Treviso',
+    'Treze de Maio',
+    'Treze Tílias',
+    'Trombudo Central',
+    'Tubarão',
+    'Tunápolis',
+    'Turvo',
+    'União do Oeste',
+    'Urubici',
+    'Urupema',
+    'Urussanga',
+    'constgeão',
+    'constgem',
+    'constgem Bonita',
+    'Vidal Ramos',
+    'Videira',
+    'Vitor Meireles',
+    'Witmarsum',
+    'Xanxerê',
+    'Xavantina',
+    'Xaxim',
+    'Zortéa',
```

Comment:
Unexpected trailing comma.

[Read more about it here.](http://eslint.org/docs/rules/comma-dangle)